libcontainer/cgroups/ebpf/devicefilter/devicefilter.go

```diff
@@ -10,8 +10,10 @@ import (
 	"math"
 	"strconv"
 
-	"github.com/cilium/ebpf/asm"
+	devicesemulator "github.com/opencontainers/runc/libcontainer/cgroups/devices"
 	"github.com/opencontainers/runc/libcontainer/devices"
+
```

Comment:
Remove the extra newline.

Or, if you want to switch to 3 groups of imports, the order should be
1. standard
2. external third-party
3. "local" (as in `goimports --local`)

IOW `github.com/opencontainers/runc/` should be the last group.

Reply:
Interesting, I've always done external third-party last (so it's in order of "locality"). But sure, I can drop the newline.